lang/jvm/src/test/scala/com/wavesplatform/lang/TypeCheckerTest.scala

```diff
@@ -147,13 +175,18 @@ class TypeCheckerTest extends PropSpec with PropertyChecks with Matchers with Sc
   private def errorTests(exprs: ((String, String), Untyped.EXPR)*): Unit = exprs.foreach {
     case ((label, error), input) =>
       property(s"Error: $label") {
-        TypeChecker(TypeCheckerContext(Map.empty,
-                                       Map.empty,
-                                       Map(
-                                         multiplierFunction.name -> multiplierFunction.signature,
-                                         noneFunc.name           -> noneFunc.signature
-                                       )),
-                    input) should produce(error)
+        TypeChecker(
+          TypeCheckerContext(
+            Map.empty,
+            Map.empty,
+            Map(
+              multiplierFunction.name -> multiplierFunction.signature,
+              noneFunc.name           -> noneFunc.signature,
+              genericFunc.name        -> genericFunc.signature
```

Comment:
Please make names more meaningful in tests too, like `idFunc`.
You can move test functon definitions where `multiplierFunction` is defined so you don't have to copy-paste the same function